### PR TITLE
header was created

### PR DESCRIPTION
**Pull request by BilokonYaroslav**
- [DEMO LINK](https://BilokonYaroslav.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://BilokonYaroslav.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,78 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a,
```
Comment (mykhalenych) [inline src/style.css:8]: use classnames instead of tag selector

File: src/style.css [hunk 2]
```diff
@@ -1 +1,78 @@
...
+  color: #000;
+}
+
+body {
+  --main-blue: #00acdc;
+  font-weight: 500;
+}
+
+.header {
+  font-family: "Roboto", sans-serif;
```
Comment (mykhalenych) [inline src/style.css:21]: move to html selector, also font-weight

File: src/style.css [hunk 3]
```diff
@@ -1 +1,78 @@
...
+a,
+a:hover,
+a:active {
+  text-decoration: none;
+  color: #000;
+}
+
+body {
+  --main-blue: #00acdc;
+  font-weight: 500;
```
Comment (mykhalenych) [inline src/style.css:17]: and delete here

**Review by mykhalenych — CHANGES_REQUESTED** [review]
Your links aren't working

**Comment by BilokonYaroslav** [comment]
Links are not working but I dont know why
Mentor right now try to help me
Have some problem with 'npm run deploy'

**Comment by BilokonYaroslav** [comment]
Links are working!

**Review by Virass — APPROVED** [review]

**Pull request closed without merging**